package.json

```diff
@@ -108,6 +108,7 @@
     "monaco-editor": "^0.45.0",
     "mri": "1.2.0",
     "path-browserify": "1.0.1",
+    "pkg-pr-new": "^0.0.5",
```

Comment:
We are using `pnpx`, do we need this dependency?

Reply:
I think it's because we want to keep the consistency of the version.

Reply:
From my understanding the X command will use the latest version though

Reply:
it should use the local one and only install the latest if a local one is not found

https://docs.npmjs.com/cli/v10/commands/npx

![image](https://github.com/QwikDev/qwik/assets/1430726/2cf0f9ab-11db-4316-a520-285fbbbbd9d7)

Reply:
Thanks for investigating into that. So keeping the dependency makes sense to me 😉